pkg/ride/compiler/stdlib/vars.json

```diff
@@ -118,6 +118,7 @@
         "UP"
       ]
     },
+    {},
```

Comment:
This is the way to add a new version. I agree this is looks weird, but otherwise we have to create a map here with lib version as a key, and handle it later in code.